sickrage/core/webserver/views.py

```diff
@@ -4381,9 +4382,9 @@ def saveProviders(self, newznab_string='', torrentrss_string='', provider_order=
                 curProvObj = sorted_providers[curProvider]
                 curProvObj.enabled = curEnabled
                 if curEnabled:
-                    providers_reordered.insert(0, curProvider)
```

Comment:
@echel0n this line means that providers which a lower preference become the most preferred when saving providers.